rust-toolchain

```diff
@@ -1 +1 @@
-stable
+1.77.0
```

Comment:
The most recent working releases were built with 1.77.2 - will this use the latest 1.77 patch?

Reply:
No; I've changed it to 1.77